### PR TITLE
Add swipe motion to main view controller

### PR DESCRIPTION
**Pull request by lucyxubroad**
## Overview
I made some optimization changes to Pear:
- add swipe motion recognizer to main screen so that you can swipe to go from Home screen to Community screen
- add a spinner loading icon to community screen before users load
- fix fade cutting off on community table view

### CODE REVIEW

**Review by camfruss — APPROVED** [review]
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

**Review by danielVebman — APPROVED** [review]
SO GOOD IT MAKES ME MAD 🤬🤬🤬
But I always find something to comment on 😎

File: Pear/Controller/TabPageViewController.swift [hunk 1]
```diff
@@ -8,17 +8,24 @@
 
 import UIKit
 
+protocol TabDelegate: class {
```
Comment (amycshuang) [inline Pear/Controller/TabPageViewController.swift:11]: can this be replaced with a closure?

**Review by amycshuang — APPROVED** [review]
SO AMAZING!!! 🤩🤩🤩🤩🚀🚀🚀🚀🚀✨✨✨✨

**Pull request merged**